src/modules/peek/Peek.FilePreviewer/FilePreview.xaml

```diff
@@ -26,6 +26,12 @@
             ToolTipService.ToolTip="{x:Bind ImageInfoTooltip, Mode=OneWay}"
             Visibility="{x:Bind IsPreviewVisible(BitmapPreviewer, Previewer.State), Mode=OneWay}" />
 
+        <Image
```

Comment:
Same as mentions earlier, I don't think we need a new Image control and Previewer we can reuse the one already available.